src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateClaimFormForSpecCallbackHandler.java

```diff
@@ -96,7 +96,7 @@ private CallbackResponse generateClaimFormForSpec(CallbackParams callbackParams)
             .build();
     }
 
-    private List<DocumentMetaData> fetchDocumentsFromCaseData(CaseData caseData, CaseDocument caseDocument) {
+    public List<DocumentMetaData> fetchDocumentsFromCaseData(CaseData caseData, CaseDocument caseDocument) {
```

Comment:
instead of making this public, I think you should move this method in a util class

Reply:
good point, I will incorporate it.